offload/plugins-nextgen/host/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ endif()
 
 # Install plugin under the lib destination folder.
 install(TARGETS omptarget.rtl.${machine}
-        LIBRARY DESTINATION "${OPENMP_INSTALL_LIBDIR}")
+        LIBRARY DESTINATION "${OFFLOAD_INSTALL_LIBDIR}")
```

Comment:
Why is this the only one that changed? In any case this will be completely gone if we land the static library patches.

Reply:
I think this should have been changed in the original offload patch, just caught it here.